deploy/common/deploy.sh

```diff
@@ -85,19 +86,23 @@ function wait_for_central {
 
     echo -n "Waiting for Central to respond."
     set +e
-    local start_time="$(date '+%s')"
+    start_time="$(date '+%s')"
+    local start_time
     local deadline=$((start_time + 10*60))  # 10 minutes
-    until $(curl_central --output /dev/null --silent --fail "https://$LOCAL_API_ENDPOINT/v1/ping"); do
```

Comment:
I think the correct fix for this is:
`until curl_central --output /dev/null --silent --fail "https://$LOCAL_API_ENDPOINT/v1/ping"; do`
i.e. remove the `$(...)`. The loop will then be terminated when `curl_central` is successful.